img/xIcon.svg

```diff
@@ -0,0 +1,3 @@
+<svg width="14" height="14" viewBox="0 0 14 14" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
[ OPTIONAL ] Please add a screenshot of the project and a live link to README so that users can view the project without actually cloning it.